android/app/src/main/java/com/google/codeu/chatme/model/User.java

```diff
@@ -74,6 +74,15 @@ public long getLastSeen() {
         return lastSeen;
     }
 
+    public void setIsOnline(boolean online) {
+        isOnline = online;
+    }
+
+    public boolean getIsOnline() {
```

Comment:
It does, `getIsOnline()` and `setIsOnline()` seem to be needed for Firebase. I'll remove `isOnline()`.